docs/topics/cache.txt

```diff
@@ -1178,45 +1178,34 @@ decorator)::
 
         return response
 
-There are a few other ways to control cache parameters. For example, HTTP
-allows applications to do the following:
-
-* Define the maximum time a page should be cached.
-
-* Specify whether a cache should always check for newer versions, only
-  delivering the cached content when there are no changes. (Some caches
-  might deliver cached content even if the server page changed, simply
-  because the cache copy isn't yet expired.)
-
-In Django, use the ``cache_control`` view decorator to specify these cache
-parameters. In this example, ``cache_control`` tells caches to revalidate the
-cache on every access and to store cached versions for, at most, 3,600 seconds::
+Generally, HTTP's caching mechanisms give you a lot of control over downstream
+caches (see :rfc:`7234` for full details). For example, even if you don't use
+Django's server-side cache framework, you can still tell clients to cache
+a view (for a certain amount of time) with the
+:rfc:`max-age <7234#section-5.2.2.8>` directive::
 
     from django.views.decorators.cache import cache_control
 
-    @cache_control(must_revalidate=True, max_age=3600)
+    @cache_control(max_age=3600)
     def my_view(request):
         # ...
 
-Any valid ``Cache-Control`` HTTP directive is valid in ``cache_control()``.
-Here's a full list:
+(If you *do* use the caching middleware, it already sets the ``max-age`` with
+the value of the :setting:`CACHE_MIDDLEWARE_SECONDS` setting. In that case,
+the custom ``max_age`` from the ``cache_control`` decorator will take
+precedence, and the header values will be merged correctly.)
+
+Any valid Cache-Control response directive is valid in ``cache_control()``.
+Here are some more examples:
 
-* ``public=True``
```

Comment:
@timgraham I removed “a full list” of arguments in favor of a few examples that illustrate the idea (how kwargs are translated into Cache-Control directives). I did this because the “full list” was not full: it matched neither the decorator (which accepts any kwargs) nor the [HTTP standards](http://www.iana.org/assignments/http-cache-directives/http-cache-directives.xhtml). I don’t think it’s a good idea to duplicate an IANA registry in Django docs.